### PR TITLE
Preserving content visibility by default (#172)

### PR DESCRIPTION
**Pull request by atanassov**
Addresses https://github.com/w3ctag/design-principles/issues/172

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/pull/240.html" title="Last updated on Jan 27, 2021, 1:50 AM UTC (c61c0ff)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/240/a264592...c61c0ff.html" title="Last updated on Jan 27, 2021, 1:50 AM UTC (c61c0ff)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -514,6 +514,19 @@ See also:
 
 This section details design principles for features which are exposed via CSS.
 
+<h3 id="css-content-should-be-visible">Content should be readable by default</h3>
```
Comment (alice) [inline index.bs:517]:
```suggestion
<h3 id="css-content-should-be-visible">Content should be perceivable by default</h3>
```

Maybe? Using WCAG language...

File: index.bs [hunk 2]
```diff
@@ -514,6 +514,19 @@ See also:
...
+
+Design CSS properties or CSS layout systems (which are typically values of the `display` property),
```
Comment (alice) [inline index.bs:519]:
```suggestion
Design CSS properties or CSS layout systems 
```

I'm not sure we need the parenthetical in this summary sentence. I think we can include it a little later on.

File: index.bs [hunk 3]
```diff
@@ -514,6 +514,19 @@ See also:
 
 This section details design principles for features which are exposed via CSS.
 
+<h3 id="css-content-should-be-visible">Content should be readable by default</h3>
+
+Design CSS properties or CSS layout systems (which are typically values of the `display` property), 
+to preserve the content as readable and usable by default.
```
Comment (alice) [inline index.bs:520]:
```suggestion
to preserve the content as perceivable and usable by default.  
```

Just being consistent. We don't need to go with this, though.

File: index.bs [hunk 4]
```diff
@@ -514,6 +514,19 @@ See also:
 
 This section details design principles for features which are exposed via CSS.
 
+<h3 id="css-content-should-be-visible">Content should be readable by default</h3>
+
+Design CSS properties or CSS layout systems (which are typically values of the `display` property), 
+to preserve the content as readable and usable by default.  
+
+<div class="example">
+For example, the default behavior of a layout system should not lead to content being clipped,
```
Comment (alice) [inline index.bs:523]: This doesn't seem like an example to me. Do we have specific examples of layout systems which demonstrate this principle?

File: index.bs (hunk 1 above)
Comment (dbaron) [inline index.bs:517]: I'd probably put this at the *end* of the CSS section rather than the beginning.

Comment (atanassov) [reply index.bs:517]: Despite 'perceivable' it's not a term used by CSS and proved to be somewhat unfamiliar for non-native speakers. The preferred title ended up being: "Content should be viewable and accessible by default".

File: index.bs (hunk 3 above)
Comment (atanassov) [reply index.bs:520]: Similar to the previous issue - this should read: "to preserve content as viewable, accessible and usable by default"

File: index.bs (hunk 1 above)
Comment (alice) [reply index.bs:517]: I can live with that.

File: index.bs (hunk 4 above)
Comment (atanassov) [reply index.bs:523]: Well, we've kept this principle true in CSSWG over the years despite not having it written out explicitly. I'll change the wording to say that: "For example, the default behavior of all layout systems in CSS will not lead to content being clipped, content overlapping other content, or content being unreachable by scrolling. "

File: index.bs (hunk 1 above)
Comment (atanassov) [reply index.bs]: Done

**Review by alice — APPROVED** [review]

**Pull request merged**